components/rmrk/Gallery/CollectionPriceChart.vue

```diff
@@ -1,160 +1,180 @@
 <template>
   <div class="price-chart mt-4">
-    <p class="label">
-      {{ $t('Price Chart') }}
-    </p>
-    <div id="chart" ref="chart" class="echart"></div>
+    <div class="is-flex is-align-items-center is-justify-content-space-between">
+      <p class="label">
+        {{ $t('Price Chart') }}
+      </p>
+      <button class="button" @click="resetZoom">Reset zoom</button>
```

Comment:
```suggestion
      <b-button type="is-primary" @click="resetZoom">Reset zoom</b-button>
```
I would change it by buefy button as I got weird behavior while button is active and :focus

Reply:
Okay, I'm on it